extensions/vault/runtime/src/main/java/io/quarkus/vault/proxy/dto/VaultCredsData.java

```diff
@@ -0,0 +1,8 @@
+package io.quarkus.vault.proxy.dto;
+
+public class VaultCredsData {
```

Comment:
Is it the same for the user pass and k8s ?

Reply:
it is not. for auth we use VaultUserPassAuth or VaultKubernetesAuth.
VaultCreds and VaultCredsData are just for the dynamic db credentials. may be it should be renamed.

Reply:
Please do it

Reply:
done